### PR TITLE
[RED-2742] Validate finetuned policy

### PR DESCRIPTION
**Pull request by jonathonsather-bc**
Script to make sure we can reproduce accuracy results on the training set. Mostly for my understanding of model I/O

### CODE REVIEW

**Review by jonathonsather-bc — COMMENTED** [review]
LGTM!
